doc/user/example_aliases2.tex

```diff
@@ -11,17 +11,17 @@
     every join/.style={link}
 ]
 
-\node[file, on chain] (s3-current) {current/m012/u.alias};
-\node[file, on chain, join] (s3-config) {config/m012/u/rx17.alias};
+\node[file, on chain] (s3-current) {current/individual/m012/u.alias};
```

Comment:
The current alias is slightly less useful these days, since the controller already figures out the band on the fly. If it also figured out the receiver serial number, it could go straight to `config/individual/m012/u/rx17-2021.alias`. But that would place more constraints on the config naming scheme.